features/non_admins_cant_merge_articles.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Merge Articles
+  As a non-admin blog user
+  I can't have special things
+  So I can't merge articles
+
+  Background:
+    Given the blog is set up
+    And an article exists with title "Entry 1" and text "This is entry 1" and author "I Like Cats"
+
+  Scenario: Non-admin cannot merge articles
```

Comment:
I think it could make sense to put this scenario in the feature in the merge_articles rather than in this separate file.